# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../euler/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -ruN euler-1.60/source/metaps.c euler-1.60-new/source/metaps.c
--- euler-1.60/source/metaps.c	2002-05-25 16:15:52.000000000 +0200
+++ euler-1.60-new/source/metaps.c	2004-06-16 23:16:49.000000000 +0200
@@ -209,7 +209,7 @@
 //	rectangle clipping
 //	x1 y1 x2 y2 setclip
 static char setclipmacro[]= "/setclip {\n\
-gsave
+gsave\n\
 	/y2 exch def\n\
 	/x2 exch def\n\
 	/y1 exch def\n\
diff -ruN euler-1.60/source/term.c euler-1.60-new/source/term.c
--- euler-1.60/source/term.c	2002-06-02 15:40:30.000000000 +0200
+++ euler-1.60-new/source/term.c	2004-06-17 09:19:46.026227976 +0200
@@ -2164,6 +2164,7 @@
 									ch = 'Û';
 									break;
 								default:
+								((void)0);
 							}
 							deadkey=0;
 							break;
@@ -2207,6 +2208,7 @@
 									ch = 'Ü';
 									break;
 								default:
+									break;
 							}
 							deadkey=0;
 							break;
@@ -2587,6 +2589,7 @@
 										ch = 'Û';
 										break;
 									default:
+									((void)0);
 								}
 								deadkey=0;
 								break;
@@ -2630,6 +2633,7 @@
 										ch = 'Ü';
 										break;
 									default:
+										break;
 								}
 								deadkey=0;
 								break;
